my_list.rb

```diff
@@ -0,0 +1,20 @@
+require './my_enumerable'
+
+class MyList
+  def initialize(list)
```

Comment:
- [ ] It would be better if you can pass multiple arguments when you initialize `MyList`. Currently, you can only pass one argument. What you are doing is pass an array as an argument, but you just pass an array as one argument.  You can fix this by just adding an asterisk `*` before the name of the argument. 

```suggestion
  def initialize(*list)
```

With this, you can pass the arguments without having to pass an array `MyList.new(1, 2, 3, 4)`